src/components/menuBar/menuBar.tsx

```diff
@@ -11,20 +12,15 @@ const MenuBarContent = styled.div`
   box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.04), 0px 2px 6px rgba(0, 0, 0, 0.04), 0px 8px 24px rgba(0, 0, 0, 0.06);
 `;
 
-const NormalButtonContainer = styled.button`
-  line-height: 17px;
-  letter-spacing: -0.02em;
-  padding: 6px 12px;
-  border-radius: 0.5rem;
-`;
-
 const SearchBar = styled.div`
   border: 1px solid #e2e7eb;
   box-sizing: border-box;
 `;
 
 export const MenuBar: React.FunctionComponent = () => {
   const [search, setSearch] = useState<string>("");
+  const [selectedButton, setSelectedButton] = useState<string>("All");
```

Comment:
Should this be of Tag type rather than string? Also All dont exist right?

Reply:
this is to track which button is being pressed right now, not the tag type. should i make an enum for it?